src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -130,7 +130,7 @@
     <value>&lt;sensitive data removed&gt;</value>
   </data>
   <data name="MSG_ExectutionCancelledKilled" xml:space="preserve">
-    <value>Process {0} was cancelled and killed.</value>
+    <value>Process {0} was killed.</value>
```

Comment:
You could fix the name of resource (typo and still refers to "cancelled"): `MSG_ExectutionCancelledKilled`